cfunits/units.py

```diff
@@ -2044,11 +2048,14 @@ def conform(cls, x, from_units, to_units, inplace=False):
     >>> Units.conform(a, Units('minute'), Units('second'))
     array([   0.,   60.,  120.,  180.,  240.])
     >>> print(a)
-    [ 0.  1.  2.  3.  4.]
-
-    >>> Units.conform(a,
-                      Units('days since 2000-12-1'),
-                      Units('days since 2001-1-1'), inplace=True)
+    [0. 1. 2. 3. 4.]
```

Comment:
I anticipate some problems with setting up doctest in a practical way for cfdm and cf-python with respect to array representation.

Namely, this case is a one-liner (as bad as it gets here in cfunits) but for higher-dimensional arrays in the other libraries the print output could easily be ugly in terms of alignment and whitespaces, and doctest doesn't seem to be able (or maybe more accurately want to try to by scope) to work out that `[0. 1. 2. 3. 4.]` is equal to `[ 0.  1.  2.  3.  4.]`, though I can get ti to ignore whitespace outside of an array `repr` by adding an option flag `doctest.NORMALIZE_WHITESPACE` to the doctest tests.